### DISCUSSION TITLE
Issue with komga regrouping under root folder name

### DISCUSSION THREAD

**Discussion by Mrcubix** [gotson/komga#1451] [Q&A] [answered]
Currently having issues while trying to make komga work for my purpose.

I have created a library on an external storage device, that is accessible by my user, mounted in /home/pi/shared/storage/komga/library.

For some reasons, when using this path, instead of komga generating an entry for each of its subfolders, it is create an entry named 'library' that only then contains the entries, althought not grouped by series.

Is there something i'm missing? I would expect komga to just work, and not consider the root folder an entry.

This is currently preventing me from reading them properly in an app such as Tachiyomi

**Comment by gotson** [comment]
From the [documentation](https://komga.org/docs/guides/scan-analysis-refresh/#what-happens-during-a-scan):

> Komga will create a Series for each subfolder of any library, whatever the depth of this subfolder in your directory structure

**Reply by Mrcubix** [reply]
Yes, and since the root folder for the created library is 'library', then any of its subfolder would be an entry in that library, which isn't the case here and the issue i'm currently having.

I repeat, it's considering 'library', as a subfolder of the root, not the root.

```
root (library)
+  Series A
      + Chapters ...
+  Series B
      + Chapters ...
+  Series C
      + Chapters ...
```
This structure appears as such inside the komga library 'Test Library' of root 'library'
```
library (root & wrong)
+  Series A (wrong because of parent)
    + Chapters ...
+  Series B (wrong because of parent)
    + Chapters ...
+  Series C (wrong because of parent)
    + Chapters ...
```
When it should appear as just
```
Series A
+  Chapters ...
Series B
+ Chapters ...
Series C
+ Chapters ...
```

Edit: Seems like even another manga reader doesn't handle this simple structure either & they aren't going to receive an update in a while, so i'm hoping the issue can be solved here in a fair amount of time.

**Reply by gotson** [reply]
you should make your way to our discord server to get support, it will be more efficient

**Reply by Mrcubix** [reply, marked as answer]
Short Version is: Komga does not support loose file, so i'll have to convert them to CBZ at some point (ComicInfo is not mandatory)
